### DISCUSSION TITLE
Golden Edition Secret Managers?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [usebruno/bruno#1921] [Q&A]
[Golden Edition](https://www.usebruno.com/pricing)

![image](https://github.com/usebruno/bruno/assets/2934507/f13a2978-ffc0-4e75-81ac-f920a84c5eb0)

Which ones are supported today?

**Comment by VictorioBerra** [comment]
Bump

**Comment by drchiodo** [comment]
also Hashecorp Vault

**Comment by sanjai0py** [comment]
We only support HashiCorp Vault at the moment and will add support for AWS Secrets Manager and Azure Key Vault in the future. Thank you for your patience.
